### PR TITLE
fix(PlannerList): bug fixes

### PR DESCRIPTION
**Pull request by sahil143**
### [ ] What does this PR do? _[Mandatory]_
Fixes issues:
- Fix Hardcoded value in toolbar
- Fix typo 'Atrributes'
- Fix Distorted Alignment of columns 

### [ ] What issue/task does this PR references? _[Mandatory]_

https://github.com/openshiftio/openshift.io/issues/3041
https://github.com/openshiftio/openshift.io/issues/3062
https://github.com/openshiftio/openshift.io/issues/3090

### [ ] Are the tests Included? _[Mandatory]_

ok Without test

### CODE REVIEW

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @sahil143 for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


For more information please head over to official [documentation](http://arquillian.org/ike-prow-plugins/#_test_keeper_plugin). You can find there how to configure the plugin - for example exclude certain file types so if PR contains only them it won't be checked.

**Comment by fabric8cd** [comment]
@sahil143 fabric8/fabric8-ui:SNAPSHOT-PR-2569-2 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2569-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@sahil143 fabric8/fabric8-ui:SNAPSHOT-PR-2569-3 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2569-fabric8-planner.badger.fabric8.io

File: src/app/components_ngrx/toolbar-panel/toolbar-panel.component.ts [hunk 1]
```diff
@@ -181,7 +194,7 @@ export class ToolbarPanelComponent implements OnInit, AfterViewInit, OnDestroy {
         } else {
           this.showSaveFilterButton = true;
         }
-      })
+      }),
```
Comment (nimishamukherjee) [inline src/app/components_ngrx/toolbar-panel/toolbar-panel.component.ts]: Do we need the comma here?

**Comment by fabric8cd** [comment]
@sahil143 fabric8/fabric8-ui:SNAPSHOT-PR-2569-4 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2569-fabric8-planner.badger.fabric8.io

**Comment by sahil143** [comment]
> if show tree is unchecked it shows total count 0

@Raunak1203 this is fixed. can you verify?

> Adding new work item is not increasing the count

This is not related to this PR. I have created an issue [here](https://github.com/fabric8-ui/fabric8-planner/issues/2575) to track this.

**Comment by fabric8cd** [comment]
@sahil143 fabric8/fabric8-ui:SNAPSHOT-PR-2569-6 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2569-fabric8-planner.badger.fabric8.io

**Comment by Raunak1203** [comment]
@sahil143 show tree issue is fixed. Could you please look at this alignment issue of distorted column if title is long https://github.com/openshiftio/openshift.io/issues/3215

**Comment by Raunak1203** [comment]
These fixes are ok without tests: for typo attributes fix and also we can't write functional tests for distorted columns fix.

**Pull request merged**